test/ruby_lsp_rails/hover_test.rb

```diff
@@ -41,21 +43,25 @@ class User < ApplicationRecord
 
           [Schema](#{URI::Generic.from_path(path: dummy_root + "/db/schema.rb")})
 
-          **id**: integer (PK)
+          ### *Columns*
```

Comment:
```suggestion
          ### Columns
```
Let's avoid italics.